scalability_and_performance/dummy-assembly.adoc

```diff
@@ -0,0 +1,23 @@
+[id="cnf-create-performance-profiles]
+= Creating a performance profile
+include::_attributes/common-attributes.adoc[]
+:context: cnf-create-performance-profiles
+toc::[]
+
+Learn about the PPC and how you can use it to create a performance profile.
+
+[NOTE]
+====
+Currently, disabling CPU load balancing is not supported by cgroup v2. As a result,
+you might not get the desired behavior from performance profiles if you have cgroup v2 enabeld. Enabling cgroup v2 is not recommended if you are using performace profiles.
+====
+
+include::modules/dummy-module-xyz.adoc[leveloffset=+1]
+
+include::modules/dummy-module-abc.adoc[leveloffset=+2]
+
+
+[role="_additional-resources"]
+= Additional Resources
```

Comment:
```suggestion
[id="additional-resources_dumy-assembly"]
= Additional Resources
```
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections 

You must provide `id` for additional resources section